<launch>
  <arg name="csv" default="path"/>
  <arg name="csv_path" default="$(find oc_demo)/scripts/$(arg csv).csv"/>
  <arg name="marker_radius" default="0.05"/>
  <arg name="start_index" default="0"/>
  <arg name="start_id" default="" />
  <arg name="manual" default="true" />
  <arg name="loop" default="true" />
  <arg name="estimate" default="true" />

  <node pkg="oc_demo" type="waypoint_navigation.py" name="waypoint_navigation" output="screen" required="true">
    <param name="csv_path" value="$(arg csv_path)" />
    <param name="start_index" value="$(arg start_index)" />
    <param name="start_id" value="$(arg start_id)" />
    <param name="manual" value="$(arg manual)" />
    <param name="linear_max" value="0.2" />
    <param name="angular_max" value="$(eval 30 * 3.141592653589793 / 180.0)" />
    <param name="angular_gain" value="1.0" />
    <param name="pose_filter" value="1" />
    <param name="loop" value="$(arg loop)"/>
    <param name="estimate" value="$(arg estimate)"/>
  </node>
</launch>
